frontends/election-manager/src/index.tsx

```diff
@@ -18,7 +18,10 @@ import { ServicesContext } from './contexts/services_context';
 const queryClient = new QueryClient({
   defaultOptions: {
     mutations: { networkMode: 'always' },
-    queries: { networkMode: 'always' },
+    queries: {
+      networkMode: 'always',
+      staleTime: Infinity,
```

Comment:
went ahead and updated the `staleTime` default in this PR, to avoid handling the `isFetching` state

Reply:
Great! Do you think it's worth leaving a comment explaining the reasoning or should we wait until we decide how to standardize this to document it?

Reply:
yeah, that makes sense - adding a comment

re: standardization: thinking we should have a shared `QueryClient` config that all apps import and use (and extend as needed), or better yet, something along the lines of a shared factory function for creating the query client

Reply:
I like both of those ideas!